src/source/Metrics/Metrics.c

```diff
@@ -58,8 +58,9 @@ STATUS getIceCandidateStats(PRtcPeerConnection pRtcPeerConnection, BOOL isRemote
 {
     STATUS retStatus = STATUS_SUCCESS;
     BOOL locked = FALSE;
-    PIceAgent pIceAgent = ((PKvsPeerConnection) pRtcPeerConnection)->pIceAgent;
-    CHK((pRtcPeerConnection != NULL || pRtcIceCandidateStats != NULL), STATUS_NULL_ARG);
+    PIceAgent pIceAgent = NULL;
+    CHK((pRtcPeerConnection != NULL && pRtcIceCandidateStats != NULL), STATUS_NULL_ARG);
+    pIceAgent = ((PKvsPeerConnection) pRtcPeerConnection)->pIceAgent;
```

Comment:
Note to self: [KvsPeerConnection.pIceAgent](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/f1073439213db4aa4b4b7cc1423609d920dd5489/src/source/PeerConnection/PeerConnection.h#L79) appears to be [guaranteed non-null](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/f1073439213db4aa4b4b7cc1423609d920dd5489/src/source/PeerConnection/PeerConnection.c#L950).